### PR TITLE
Documentation about cli

### PR DESCRIPTION
**Pull request by ankitjena**

### CODE REVIEW

File: docs/datamodel.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Example
+
```
Comment (wtrocki) [inline docs/datamodel.md]: little bit raw. Requires some sentence / introduction what this chapter is about

File: docs/usage.md [hunk 2]
```diff
@@ -1,71 +1,37 @@
...
-This will create a separate table `user_note` to track the relationship using columns `userId` and `noteId`. You can customize the table name as such:
-```graphql
-type Note {
-  users: [User!]! @ManyToMany(tablename: "yourCustomTable")  
-}
-```
-See ./testss for use cases and tests on relations.
-git
\ No newline at end of file
+```bash
```
Comment (wtrocki) [inline docs/usage.md]: very raw. Lets add some sentence what this project is doing etc. and actual getting started

File: docs/usage.md [hunk 3]
```diff
@@ -1,71 +1,37 @@
...
+## Commands
+### Init
+  ```bash
+  graphback init <project-name>
+  ```
+  The command will guide you through a series of questions - choosing a template, bootstraping model - and setup your project for you.
+
+  Example:
+
+  ![](init.png)
```
Comment (wtrocki) [inline docs/usage.md]: use images folder

**Comment by wtrocki** [comment]
I think we are missing getting started. I am not possible to get started now by reading the docs. We completely ignored what is required (docker-compose) and what is the flow of commands you should start with.

**Comment by ankitjena** [comment]
@wtrocki added them. Let's refactor this to docusaurus in another PR.

**Pull request merged**